minifi-bootstrap/pom.xml

```diff
@@ -45,5 +45,16 @@ limitations under the License.
             <groupId>org.apache.nifi</groupId>
             <artifactId>nifi-expression-language</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.yaml</groupId>
```

Comment:
Snakeyaml is licensed using an apache 2.0 license and does not include a notice[1]. It is my understanding that you don't need to do anything if both those are true.

[1] https://bitbucket.org/asomov/snakeyaml/src